op.c

```diff
@@ -699,8 +699,6 @@ S_bad_type_pv(pTHX_ I32 n, const char *t, const OP *o, const OP *kid)
 		 (int)n, PL_op_desc[(o)->op_type], t, OP_DESC(kid)), 0);
 }
 
-/* remove flags var, its unused in all callers, move to to right end since gv
```

Comment:
Because it describes what was being changed in the commit it was added to, referring to a function parameter that has been removed, and should have been in the commit message.